src/store.c

```diff
@@ -22,44 +22,9 @@
 #include "store_ro_http_proxy.h"
 #include "store_ro_composite.h"
 #include "store_null.h"
+#include "g_logger.h"
 
-//TODO: Make this function handle different logging backends, depending on if on compiles it from apache or something else
```

Comment:
I think I may have initially misunderstood this TODO comment, the `g_logger` function implementation does pretty much the same as this `log_message` function did. It  does not, however, implement what the TODO describes, I.E. calling `ap_log_*` functions when built into `mod_tile.so`.